src/ui/forms/_forms.scss

```diff
@@ -1,5 +1,6 @@
 input[type="number"] {
   max-width: 100px;
+  -moz-appearance:textfield;
```

Comment:
Perhaps we should move the `-webkit-appearance` property up here too. Does that break webkit?

Also we should add a non-vendor-prefix style too.